File: Count Odd and Even/Count_Odd_and_Even.cpp
class Solution {
  public:
    pair<int, int> countOddEven(vector<int> &arr) {
        int odd = 0,even = 0;
        int n = arr.size();
        for(int i = 0; i<n; i++) {
            if(arr[i]%2 == 1) odd++;
            else even++;
        }
        return {odd,even};
    }
};

File: Count Odd and Even/Count_Odd_and_Even.java
class Solution {
    public int[] countOddEven(int[] arr) {
        int odd = 0,even = 0;
        int n = arr.length;
        for(int i = 0; i<n; i++) {
            if(arr[i]%2 == 0) even++;
            else odd++;
        }
        return new int[]{odd,even};
    }
}